haproxy/datadog_checks/haproxy/haproxy.py

```diff
@@ -81,6 +81,8 @@ def __init__(self, name, init_config, instances):
         # https://en.wikipedia.org/wiki/Autovivification
         # https://gist.github.com/hrldcpr/2012250
         self.host_status = defaultdict(lambda: defaultdict(lambda: None))
+        self.tags_regex = self.instance.get('tags_regex')
```

Comment:
Shall we also use `self._tag_from_regex(service)` 

here https://github.com/DataDog/integrations-core/blob/aae1bd2a95733bffaef4fec8b9d4b6e797443719/haproxy/datadog_checks/haproxy/haproxy.py#L846 ?

Reply:
The call to `_process_service_check` and `_process_metrics` have a `custom_tags` parameter (`line_tags`) that already includes regex tags. The difference with status metrics is they're not in the `data` for loop

https://github.com/DataDog/integrations-core/blob/aae1bd2a95733bffaef4fec8b9d4b6e797443719/haproxy/datadog_checks/haproxy/haproxy.py#L400-L402

Reply:
Oh, I see. Thx.